.circleci/config.yml

```diff
@@ -37,3 +40,4 @@ workflows:
               only:
                 - master
                 - production
+                - /^(?!pull\/).*$/
```

Comment:
what does this syntax mean?

Reply:
That is a regular expression, it tells CircleCI to only build master, production or pull requests only.